skins/laika/src/components/App.vue

```diff
@@ -3,7 +3,15 @@
 		<header>
 			<Header :page-identifier="pageIdentifier" :assets-path="assetsPath"></Header>
 		</header>
-		<main id="app" class="section">
+		<main id="app" class="main-wrapper">
+			<div class="container">
+				<div class="columns has-margin-top-36 has-margin-bottom-36 intro-content">
+					<div class="column is-two-thirds is-half-desktop has-padding-0">
+						<span>Die Wikimedia Fördergesellschaft ist eine unabhängige gemeinnützige Organisation,
+							die in Deutschland Spenden für Wikipedia und andere Wikimedia-Projekte sammelt.</span>
+					</div>
```

Comment:
This translation has slipped in an otherwise well-translated PR.

Reply:
Will be taken care of in a follow up PR.